modules/custom/az_core/src/Plugin/ConfigProvider/QuickstartConfigProvider.php

```diff
@@ -118,8 +118,23 @@ public function addInstallableConfig(array $extensions = []) {
       $data = $profile_storage->readMultiple(array_keys($data)) + $data;
     }
 
+    // Get permissions defined.
+    // @phpstan-ignore-next-line
+    $permission_definitions = \Drupal::service('user.permissions')->getPermissions();
```

Comment:
Use injection from container if possible. (These ConfigProvider classes are a little oddly constructed)

Reply:
I've created a follow-up issue for this: #1558

Reply:
Thanks @joeparsons ! :+1: